src/h5web/toolbar/controls/DomainSlider/Histogram.tsx

```diff
@@ -52,13 +58,20 @@ function Histogram(props: Props) {
             x={xScale(bins[i])}
             y={height - yScale(d)}
             width={xScale(bins[i + 1]) - xScale(bins[i]) + 0.5} // +0.5 removes the small gap between bars
-            height={yScale(d)}
+            height={yScale(d) - yScale(0)}
```

Comment:
This was not needed before as `yScale(0)` was equal to `0` but this is no longer true if there is a colorbar.